### ISSUE TITLE
Crate deployment test

### ISSUE THREAD

**Issue by alexjpwalker** [typedb/typedb-driver#488]
We need to implement `test-deployment-crate` in CI to verify correctness of the distribution.

**Comment by farost** [comment]
Correct. Closing this as a duplicate in favor of a more general test coverage issue https://github.com/typedb/typedb-driver/issues/512
